lib/text_sender.rb

```diff
@@ -0,0 +1,20 @@
+require 'rubygems'
+# require 'twilio-ruby'
+
+module TextSender
+  # def time
+  #   time_now_add_one_hour = Time.now + 60*60
+  #   time_now_add_one_hour.strftime ('%H:%M')
+  # end
+
+  # def send_text
+  #   account_sid = 'AC13374efe61d64fefa542d1c60e029ba4'
+  #   aouth_token = '{{df0375dec773ce13f60f188637bdf9d2}}'
+  #   @client = Twilio::REST::Client.new account_sid, aouth_token
+  #   sms = @client.account.sms.messsages.create(:body => "Thanks for your order. It will be delivered at #{time}.",:to => "447425529008",
```

Comment:
Metrics/LineLength: Line is too long. [138/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)